src/com/company/Main.java

```diff
@@ -59,10 +59,11 @@ public static void checkforexistance(File file) {
                 System.out.println("file doesn't found ");
             }
         }
-        public static void avoidspecialchar(String data) {
+        public static void avoidSpecialChar(String data)
+        {
             data = data.replaceAll("[^a-zA-Z0-9@]", " ");
```

Comment:
It is always better to return the modified data, instead of actually modifying the reference inside the method.

Reply:
yeah i have made this change in  recently updated code.